apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/entity/blockchain/TransactionImpl.java

```diff
@@ -752,6 +490,9 @@ public boolean equals(Object o) {
 
     @Override
     public byte[] getSenderPublicKey() {
+        if (senderPublicKey == null) {
```

Comment:
I'm not sure, may be incorrect behavior?

Reply:
Its ok, exception should not be thrown, but just in case i put it here to signal about inconsistency